lib/buffer-search.coffee

```diff
@@ -58,6 +59,17 @@ class BufferSearch
       ResultsMarkerLayersByEditor.set(editor, layer)
     layer
 
+  checkSanity: (findPattern) ->
```

Comment:
I'd prefer this method be named a bit more precisely. How about `doesMatchTheEmptyString(findPattern)`?